modules/widget-visibility/widget-conditions.php

```diff
@@ -142,8 +142,12 @@ public static function widget_conditions_options_echo( $major = '', $minor = ''
 				?>
 				<option value=""><?php _e( 'All taxonomy pages', 'jetpack' ); ?></option>
 				<?php
-
-				$taxonomies = get_taxonomies( array( '_builtin' => false ), 'objects' );
+				/**
+				 * Filters args passed to get_taxonomies.
+				 * @see https://developer.wordpress.org/reference/functions/get_taxonomies/
+				 * @param array jetpack_widget_visibility_tax_args
+				 */
+				$taxonomies = get_taxonomies( apply_filters( 'jetpack_widget_visibility_tax_args', array( '_builtin' => false ) ), 'objects' );
```

Comment:
Could you add line breaks to make the whole line a bit more readable, as well as more information (`@since` and `@module`) in the inline docs, as shown [here](https://github.com/Automattic/jetpack/blob/master/modules/widget-visibility/widget-conditions.php#L398-L404)?

You can use `4.4.0` as the `@since` version.

Thanks!